vmm/src/interrupt.rs

```diff
@@ -4,7 +4,8 @@
 //
 
 use devices::interrupt_controller::InterruptController;
-
+#[cfg(target_arch = "aarch64")]
+use hypervisor::kvm::KVM_MSI_VALID_DEVID;
```

Comment:
Can this `use` clause be moved into `pub mod kvm` below? Thanks.

Reply:
Yes, it was in wrong place. Thanks.